lib/katello_attach_subscription/utils.rb

```diff
@@ -14,6 +14,25 @@ def self.search_args(search)
         search
       end
     end
+
+    def self.merge_subs(current_sub, sub_to_merge, command)
+      if not current_sub.nil?
+        current = current_sub.clone
+      else
+        current = {}
+      end
+      new_sub = sub_to_merge.clone
+      case command
+      when "override"
+        current = new_sub
+      else
+        if current.nil?
```

Comment:
when is this true? I'd think that `current` is either `{}` or `current_sub`

Reply:
maybe it's better `if current.empty?`
or better
`current.merge!(new_sub)` works as `current = new_sub` if `current` is `{}`?